### PR TITLE
cli: fix changefeed-id need to be limit max length 

### PR DESCRIPTION
**Pull request by ben1009**
### What problem does this PR solve? 
fix `changefeed-id need to be limit max length` #1709

### What is changed and how it works?
add a limit check

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

Code changes

 - Add changfeed-id limit check

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### CODE REVIEW

File: cdc/kv/etcd_test.go [hunk 1]
```diff
@@ -471,10 +471,7 @@ func (s *etcdSuite) TestCreateChangefeed(c *check.C) {
 		SinkURI: "root@tcp(127.0.0.1:3306)/mysql",
 	}
 
-	err := s.client.CreateChangefeedInfo(ctx, detail, "bad.id👻")
```
Comment (liuzix) [inline cdc/kv/etcd_test.go:474]: Why is this case deleted?

Comment (ben1009) [reply cdc/kv/etcd_test.go:474]: https://github.com/pingcap/ticdc/pull/2054/files#diff-cc1de9ecd51350251a4837f8c67b15b653ed937eedf6cc61daebdf28644bb973L301
move the validation to `cmd/client_changefeed.go`, validate first
https://github.com/pingcap/ticdc/pull/2054/files#diff-75cf14b54b555126511c6140e14beb602a4c7a0ccec637a7996e9d2cf256c6dcR462

**Review by overvenus — APPROVED** [review]
LGTM

**Review by liuzix — APPROVED** [review]

**Pull request merged**